WcaOnRails/config/routes.rb

```diff
@@ -78,7 +78,12 @@
     get '/admin/check-existing-results' => "admin#check_results", as: :admin_check_existing_results
     post '/admin/upload-json' => "admin#create_results", as: :admin_upload_results
     post '/admin/clear-submission' => "admin#clear_results_submission", as: :clear_results_submission
+    get '/admin/post-results' => 'admin#post_results', as: :admin_post_results
```

Comment:
If the routes changes something in the db, it definitely shouldn't be a get: most of the routes here should be `post` or `delete` I think.

You would still be able to use a simple `link_to` to link to them though, by passing `method: :delete`/`method: :post`.

Reply:
Cool, done!